.env

```diff
@@ -1,4 +1,4 @@
 WESTEGG_URL=http://localhost:3001
 NEXT_PUBLIC_WESTEGG_URL=/api
 NEXT_PUBLIC_WESTEGG_VERSION=v1
-NEXT_PUBLIC_BOOTSTRAP_NODES=
+NEXT_PUBLIC_BOOTSTRAP_NODES=/ip4/staging.gatsby.sh/tcp/8081/ws/ipfs/12D3KooWC3FCg8mepBicz1pFxRyUzQR5rvrjDHRKFXr1cP2dfZmL,/dns4/staging.gatsby.sh/tcp/8085/wss/ipfs/QmSmFrQJFsQ312RqJQXBjQTRiUgWnubfPbCeX1g6PbJRUQ
```

Comment:
The first address doesn't look like a valid multiaddress since it is using `ip4` with a domain name. Are you sure you can connect to that address?